### DISCUSSION TITLE
Processor feature

### DISCUSSION THREAD

**Discussion by padmohan-clgx** [open-telemetry/opentelemetry-collector#8669] [Q&A] [answered]
Is any processors features available for masking data?

**Comment by spadger** [comment, marked as answer]
Yes, depending on what exactly you need, and your attitude towards risk
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor

**Reply by TylerHelmuth** [reply]
Also https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/redactionprocessor
